### PR TITLE
patch to Cosmos v0.44.0

### PR DESCRIPTION
**Pull request by Benzbeeb**
## Implementation details

- (bump) Use cosmos-sdk package v0.44.0
- (bump) Use ibc-go package v1.1.0

## Please ensure the following requirements are met before submitting a pull request:

- [x] The pull request is targeted against the correct target branch
- [x] The pull request is linked to an issue with appropriate discussion and an accepted design OR is linked to a spec that describes the work.
- [x] The pull request includes a description of the implementation/work done in detail.
- [ ] The pull request includes any and all appropriate unit/integration tests
- [x] You have added a relevant changelog entry to `CHANGELOG_UNRELEASED.md`
- [ ] You have re-reviewed the files affected by the pull request (e.g. using the `Files changed` tab in the Github PR explorer)

### CODE REVIEW

**Review by songwongtp — APPROVED** [review]

**Review by p-rial — APPROVED** [review]

File: go.sum [hunk 1]
```diff
@@ -203,8 +203,9 @@ github.com/coreos/go-systemd v0.0.0-20190321100706-95778dfbb74e/go.mod h1:F5haX7
 github.com/coreos/go-systemd/v22 v22.3.2/go.mod h1:Y58oyj3AT4RCenI/lSvhwexgC+NSVTIJ3seZv2GcEnc=
 github.com/coreos/pkg v0.0.0-20160727233714-3ac0863d7acf/go.mod h1:E3G3o1h8I7cfcXa63jLwjI0eiQQMgzzUDFVpN/nH/eA=
 github.com/coreos/pkg v0.0.0-20180928190104-399ea9e2e55f/go.mod h1:E3G3o1h8I7cfcXa63jLwjI0eiQQMgzzUDFVpN/nH/eA=
-github.com/cosmos/cosmos-sdk v0.43.0 h1:l2GXJMDVtJyHb35pDUCw+uyr6eZtBo8vt+7PSsq+Fjo=
 github.com/cosmos/cosmos-sdk v0.43.0/go.mod h1:ctcrTEAhei9s8O3KSNvL0dxe+fVQGp07QyRb/7H9JYE=
```
[the remark this answers is not in the document]
Comment (taobun) [reply go.sum]: It's come from ibc-go :D

**Review by taobun — CHANGES_REQUESTED** [review]
LGTM

**Review by taobun — APPROVED** [review]

**Review by Kissadada — APPROVED** [review]
LGTM

**Review by taobun — CHANGES_REQUESTED** [review]
Please update changelog and PR description

File: CHANGELOG.md [hunk 2]
```diff
@@ -2,6 +2,8 @@
 
 ## [v2.2.0](https://github.com/bandprotocol/chain/releases/tag/v2.2.0)
 
+- (bump) Use cosmos-sdk package v0.44.0
+- (bump) Use ibc-go package v1.1.0
```
Comment (taobun) [inline CHANGELOG.md:6]: Move to v2.3.0

**Review by taobun — CHANGES_REQUESTED** [review]

**Review by taobun — APPROVED** [review]
LGTM

**Pull request merged**